src/phoenix/config.py

```diff
@@ -424,9 +424,9 @@ def from_env(cls, idp_name: str) -> "OAuth2ClientConfig":
                 f"An OpenID Connect configuration URL must be set for the {idp_name} OAuth2 IDP "
                 f"via the {oidc_config_url_env_var} environment variable"
             )
-        if urlparse(oidc_config_url).scheme != "https":
+        if urlparse(oidc_config_url).scheme != "https" and urlparse(oidc_config_url).hostname != "localhost":
```

Comment:
```suggestion
        parsed_oidc_config_url = urlparse(oidc_config_url)
        is_local_oidc_config_url = parsed_oidc_config_url.hostname in ("localhost", "127.0.0.1")
        if parsed_oidc_config_url.scheme != "https" and not is_local_oidc_config_url:
```

Let's check for both `127.0.0.1` and `localhost`

Reply:
Thank you for the review. I also thaught about that, but then somehow didn't decided to add it.